packages/export/src/nodes/pivot.js

```diff
@@ -0,0 +1,176 @@
+import _ from 'lodash/fp.js'
+import F from 'futil'
+import ContextureClient from 'contexture-client'
+import {
+  getKey,
+  resultsForDrilldown,
+} from 'contexture-client/exampleTypes/pivot.js'
+import { andGroup, runWith, setFilterOnly } from '../utils.js'
+
+// TODO move to shared folder
+let getPageSize = (grouping) => {
+  let pageSizeGetters = {
+    fieldValues: _.getOr(10, 'size'),
+  }
+  return _.getOr(_.noop, grouping.type, pageSizeGetters)(grouping)
+}
+
+export let getGroupingSize = (
+  node,
+  groupingType,
+  cardinalityResult,
```

Comment:
There are lots of references here to `cardinality`, which is only correct for fieldValues. Now that the elasticsearch provider handles other cases, we should match the terminology